src/app/shared/utilities/view.utility.ts

```diff
@@ -160,21 +160,47 @@ const equals = (filterOne: Filter, filterTwo: Filter): boolean => {
   return filterTwo ? filterOne.field === filterTwo.field && filterOne.value === filterTwo.value : true;
 };
 
+/**
+ * Traverse object by path returning value else undefined.
+ *
+ * {
+ *   'trainee': {
+ *     'label': 'Name of organization'
+ *   }
+ * }
+ * i.e. `trainee.label` returns 'Name of organizarion'
```

Comment:
I'm hesitant to have complex structures in javadocs.
It can be helpful but we probably should utilize the fancy javadoc functionality to designate this as code, preformat, or whatever is available.
(I say javadoc because that appears to be the format being followed. Does this javadoc style documentation?)

I think this should be something to look into.
Is `<code>` or `<markup>` available?

Examples ([from this URL](https://reflectoring.io/howto-format-code-snippets-in-javadoc/)):
```
/**
 * Using {@code @code} alone, indentation will be lost, but you don't have to
 * escape special characters:
 * 
 * {@code An annotation <code>@Foo</code>; and a generic List<String>}.
 */
...
/**
 * <pre>{@code 
 * public class JavadocTest {
 *   // indentation and line breaks are kept 
 * 
 *  @literal @SuppressWarnings
 *   public List<String> generics(){
 *     // '<' and '>'  are displayed correctly
 *     // '@' CANNOT be escaped with HTML code, though!
 *   }
 * } 
 * }</pre>
 */
```

Reply:
These are actually not JavaDocs but intended to eventually be TypeDocs. I am not sure yet the specific syntax and sturcture for TypeDocs. Just thought I would give it a try writing comments with code. :)

Reply:
https://typedoc.org/guides/doccomments/

Reply:
I am going to leave this be for now.

Reply:
> I am going to leave this be for now.

Sounds like a good idea.
I am not very familiar with TypeDocs.
They look very similar to Github's and Slacks, such as the use of three backticks for code blocks.

I'll resolve this comment as we need more investigation and planning to determine the style guide to follow and use.